src/base/bittorrent/torrenthandle.h

```diff
@@ -315,6 +316,14 @@ namespace BitTorrent
         void setName(const QString &name);
         void setSequentialDownload(bool b);
         void toggleSequentialDownload();
+        // disables deadlines if completionTime < 0
+        void scheduleDownloading(std::chrono::milliseconds totalDeadline,
+                                 std::chrono::milliseconds delay = std::chrono::milliseconds(0),
```

Comment:
Can we have here `std::chrono::milliseconds delay = {0}`?
Or even `std::chrono::milliseconds delay = {}`?